clients/privacy-center/components/consent/NoticeDrivenConsent.tsx

```diff
@@ -232,6 +232,7 @@ const NoticeDrivenConsent = () => {
     const consentCookieKey: CookieKeyConsent = Object.fromEntries(noticeKeyMap);
     window.Fides.consent = consentCookieKey;
     const updatedCookie = { ...cookie, consent: consentCookieKey };
+    updatedCookie.fides_meta.consentMethod = ConsentMethod.SAVE; // include the consentMethod as extra metadata
```

Comment:
This is the one-liner on the Privacy Center to record the consent method as well.